EIPS/eip-2930.md

```diff
@@ -91,6 +91,8 @@ def process_access_list(access_list) -> Tuple[List[Set[Address], Set[Pair[Addres
 
 The access list is NOT charged per-byte fees like tx data is; the per-item costs described above are meant to cover the bandwidth costs of the access list data in addition to the costs of accessing those accounts and storage keys when evaluating the transaction.
 
+Additionally, the transaction is valid only if `chainId` is equal to the chain id of the current network.
```

Comment:
```suggestion
Clarification: the transaction is valid only if `chainId` is equal to the chain id of the current network. 
```
That sentence is now more of a clarification, it doesn't really _add_ anything anymore